rocky/OOI_database_seed.json

```diff
@@ -225,7 +225,7 @@
     "fields": {
       "last_updated": "2021-08-09T14:50:09.835Z",
       "data": {
-        "description": "The website does not use HTTP Strict Transport Security (HSTS). Via HSTS the browser is configured to only connect via a proper secured HTTPS connection to the website.",
+        "description": "The website does not use HTTP Strict Transport Security (HSTS). HSTS tells the browser to connect via a proper secured HTTPS connection to the website.",
```

Comment:
HSTS tells to browser to only conect over https, and remember this for future visits, possibly including subdomains.